.circleci/config.yml

```diff
@@ -216,4 +329,6 @@ workflows:
       - discovery_integration_test
       - chatgpt_api_integration_test_mlx
       - test_macos_m1
+      - build_exo_macos
+      - build_exo_linux
       # - chatgpt_api_integration_test_tinygrad
```

Comment:
why not `build_exo_windows`?

Reply:
do you know which dependencies are problematic right now?
tinygrad supports windows fully now so we should be able to get exo working on windows with little effort